deblur/workflow.py

```diff
@@ -41,9 +47,95 @@ def dereplicate_seqs(seqs_fp,
         log_name=log_name)
 
 
-def remove_artifacts_seqs(seqs_fp):
-    """Step 3: remove artifacts from FASTA file"""
-    pass
+def remove_artifacts_seqs(seqs_fp,
+                          ref_fp,
+                          output_fp,
+                          ref_db_fp=None,
+                          negate=False,
+                          threads=1):
+    """Step 3: remove artifacts from FASTA file using
+       SortMeRNA
+
+    Parameters
+    ----------
+    seqs_fp: string
+        file path to FASTA input sequence file
+    ref_fp: tuple
```

Comment:
That is just shifting that "expensive" step outside the workflow, but for a single interaction will be same...
Probably I am assuming something that is not correct. Does index db over 100 files of 1 seq perform exactly equal than index db in a single file with 100 seqs? If not, we may want to discourage that behavior.